### ISSUE TITLE
Multi Org Phase 2 - Filter deactivated organizations from organization dropdown

### ISSUE THREAD

**Issue by wdoconnell** [influxdata/ui#5935]
AIM will modify the /quartz/accounts/:accountID/orgs endpoint that returns the orgs in the user's current account to also return orgs currently in the process of being deleted, with a property that identifies the deletion state.

Current logic in global header lists _all_ orgs returned from this endpoint when rendering the list of orgs in the user's current account. This will need to be adjusted when the endpoint changes so that the header doesn't display - and the user can't try to switch to - a deleted org.

**Comment by ALuckette** [comment]
On hold, waiting for the deactivated flag from the AIM team.

**Comment by hoorayimhelping** [comment]
Blocked on backend property to get deactivation status
